src/cdomains/lval.ml

```diff
@@ -105,7 +111,7 @@ struct
     | `Index(i,o) -> NoOffset (* array domain can not deal with this -> leads to being handeled as access to unknown part *)
 end
 
-module Offset (Idx: IntDomain.Z) =
+module Offset (Idx: IdxDomain) =
```

Comment:
It's nice that this can be weakened from `IntDomain.Z`. I was afraid we might need a full `IntDomain.Z` implementation for unit.